app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
```suggestion
     @wraps(func)
    def wrapper(*args) -> Callable:
```
Good to add annotation, that this is a wrapped function.